services/avsregistry/avsregistry_chaincaller_test.go

```diff
@@ -144,15 +140,85 @@ func TestAvsRegistryServiceChainCaller_GetOperatorsAvsState(t *testing.T) {
 			service := NewAvsRegistryServiceChainCaller(mockAvsRegistryReader, mockElReader, mockPubkeyCompendium, logger)
 
 			// Call the GetOperatorPubkeys method with the test operator address
-			gotOperatorsAvsStateDict, aggG1PubkeyPerQuorum, gotErr := service.GetOperatorsAvsStateAtBlock(context.Background(), tt.queryQuorumNumbers, tt.queryBlockNum)
+			gotOperatorsAvsStateDict, gotErr := service.GetOperatorsAvsStateAtBlock(context.Background(), tt.queryQuorumNumbers, tt.queryBlockNum)
 			if tt.wantErr != gotErr {
 				t.Fatalf("GetOperatorsAvsState returned wrong error. Got: %v, want: %v.", gotErr, tt.wantErr)
 			}
 			if tt.wantErr == nil && !reflect.DeepEqual(tt.wantOperatorsAvsStateDict, gotOperatorsAvsStateDict) {
 				t.Fatalf("GetOperatorsAvsState returned wrong operatorsAvsStateDict. Got: %v, want: %v.", gotOperatorsAvsStateDict, tt.wantOperatorsAvsStateDict)
 			}
-			if tt.wantErr == nil && !reflect.DeepEqual(tt.wantAggG1PubkeyPerQuorum, aggG1PubkeyPerQuorum) {
-				t.Fatalf("GetOperatorsAvsState returned wrong aggG1PubkeyPerQuorum. Got: %v, want: %v.", aggG1PubkeyPerQuorum, tt.wantAggG1PubkeyPerQuorum)
+		})
+	}
+}
+
+func TestAvsRegistryServiceChainCaller_GetQuorumsAvsState(t *testing.T) {
+	logger := logging.NewNoopLogger()
+	testOperator := testOperator{
+		operatorAddr: common.HexToAddress("0x1"),
+		operatorId:   types.OperatorId{1},
+		pubkeys: types.OperatorPubkeys{
+			G1Pubkey: bls.NewG1Point(big.NewInt(1), big.NewInt(1)),
+			G2Pubkey: bls.NewG2Point([2]*big.Int{big.NewInt(1), big.NewInt(1)}, [2]*big.Int{big.NewInt(1), big.NewInt(1)}),
+		},
+	}
+
+	var tests = []struct {
```

Comment:
Do you need to add any error test cases?

Reply:
Probably should eventually, but no time right now. Added TODO in: [9066456](https://github.com/Layr-Labs/eigensdk-go/pull/24/commits/90664564eb4cb69cc1b500917defdf090c84e6d4)